packages/slate-react/src/utils/find-dom-range.js

```diff
@@ -20,8 +20,8 @@ function findDOMRange(range) {
   const r = window.document.createRange()
   const start = isBackward ? focus : anchor
   const end = isBackward ? anchor : focus
-  r.setStart(start.node, start.offset)
-  r.setEnd(end.node, end.offset)
+  r.setStart(start.node, start.offset >= 0 ? start.offset : 0)
+  r.setEnd(end.node, end.offset >= 0 ? end.offset : 0)
```

Comment:
Hey @RobertDaleSmith can you fix these inside `findDOMPoint` instead so that it doesn't return `-1` in these cases, instead of at this layer above? I think that will be the most bulletproof for the future. Thanks!